src/extension/vectortiles/pom.xml

```diff
@@ -25,7 +25,6 @@
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
-      <version>2.3.1</version>
```

Comment:
~~Why is version removed here?~~

Nevermind, I see it is moved up to `src/pom`